tooling/minimize/src/bb.rs

```diff
@@ -272,7 +272,26 @@ impl<'cx, 'tcx> FnCtxt<'cx, 'tcx> {
 
                 return TerminatorResult { stmts: list!(stmt), terminator };
             }
+            rs::sym::size_of_val => {
+                let destination = self.translate_place(&destination, span);
+
+                let size = self.size_of(intrinsic.args.type_at(0));
```

Comment:
For unsized types this won't be right. We can't make this a constant. So it makes little sense to translate this as a constant IMO -- we should add new operands for this in MiniRust proper.